### DISCUSSION TITLE
Trigger Automation on request

### DISCUSSION THREAD

**Discussion by s1e2b3i4** [hassio-addons/addon-nginx-proxy-manager#233] [Q&A] [answered]
Is there any possibility to trigger an automation on a specific request to the proxy?
I would really love to wake up my media server via WOL if I request a service like Plex on it through the proxy.

If this add-on is not supposed to do such a thing, one other idea I have is to expose the `access.log` file to the host. With that one could use the [File Sensor](https://www.home-assistant.io/integrations/file/#sensor) to filter the log file a use that as a trigger for an automation. 

Thanks in advance and best regards,
Sebastian

**Comment by frenck** [comment, marked as answer]
Your question is not related to this add-on.

Nevertheless, Home Assistant has webhook support, you might want to take a look at that.

**Reply by s1e2b3i4** [reply]
Thanks for the quick reply! I will look into that.
